components/count-badge/count-badge.js

```diff
@@ -0,0 +1,128 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class CountBadge extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * The number to be displayed on the badge. If badge-type is "notification", the number will be truncated to three digits ("99+").
```

Comment:
This should be two digits. I've gone by the maximum number of digits in the number portion, and not including the + sign.